src/test/scala/org/constellation/rollback/RollbackCalculateBalancesGenesisTest.scala

```diff
@@ -16,20 +17,19 @@ class RollbackCalculateBalancesGenesisTest
   implicit val dao: DAO = TestHelpers.prepareRealDao()
 
   private val rollbackAccountBalances: RollbackAccountBalances = new RollbackAccountBalances
-  private val distributionTransactionAmount = 100000000000000L
-  private val genesisTransactionAmount = 400000000000000000L
-
-  private val genesisKeyPair = KeyUtils.makeKeyPair()
-  private val tipKeyPair = KeyUtils.makeKeyPair()
-  private val genesisObservation = Genesis.createGenesisAndInitialDistributionDirect(
-    genesisKeyPair.getPublic.toId.address,
-    Set(tipKeyPair.getPublic.toId),
-    genesisKeyPair
+  private val initialBalances = Seq(
+    AccountBalance("a", 10),
+    AccountBalance("b", 20),
+    AccountBalance("c", 30),
+    AccountBalance("d", 50)
   )
 
+  private val genesisObservationWithInitialBalances = Genesis.createGenesisObservation(initialBalances)
+  private val emptyGenesisObservation = Genesis.createGenesisObservation(Seq.empty)
+
   "Genesis Observation" - {
```

Comment:
@mchrapek please take a look if these tests are adopted to the logic in a correct way